shared/lib-hoppy/hoppy/config.py

```diff
@@ -5,6 +5,7 @@
     "username": os.environ.get("RABBITMQ_PLACEHOLDERS_USERNAME") or "guest",
     "password": os.environ.get("RABBITMQ_PLACEHOLDERS_USERPASSWORD") or "guest",
     "port": int(os.environ.get("RABBITMQ_PORT") or 5672),
+    "virtual_host": os.environ.get("RABBITMQ_VHOST") or "/",
```

Comment:
My feeling is that this variable is more similar to something like port number than username/password, i.e. in the vast majority of cases, you shouldn't have to change it. I'm hesitant to bring it into `setenv.sh` and require it in docker files, etc. but I'm open to discussing it!

Reply:
That's True! I agree that it wont change much! I'm not super tied to the name either! I do think that you should add it to the docker files at least, and if the local env does't have the var set then it will behave as if it weren't there. I dont think you need to add it to setenv.sh at all. But i could be wrong about how they will behave once its deployed via helm. 🤷